swift/aperture/main.swift

```diff
@@ -75,7 +75,7 @@ if numberOfArgs == 8 {
 }
 
 if numberOfArgs == 2 && CommandLine.arguments[1] == "list-audio-devices" {
-  print(try! toJson(DeviceList.audio()))
+  print(try toJson(DeviceList.audio()))
```

Comment:
Was a bit unsure about using `try` in global scope outside of a do/catch block, but I tested it in a playground and it does work 👍